imap_processing/ialirt/utils/create_xarray.py

```diff
@@ -144,7 +144,17 @@ def create_xarray_from_records(records: list[dict]) -> xr.Dataset:  # noqa: PLR0
     # Populate the dataset variables
     for i, record in enumerate(records):
         for key, val in record.items():
-            if key in ["apid", "met", "met_in_utc", "ttj2000ns", "last_modified"]:
+            if key in [
+                "apid",
+                "met",
+                "met_in_utc",
+                "ttj2000ns",
+                "last_modified",
+                "sc_position_GSM",
```

Comment:
I thought they did want the positions in the CDFs? They only want them in the web queries instead?